packages/expo-dev-menu/metro.config.js

```diff
@@ -2,6 +2,23 @@ const { createMetroConfiguration } = require('expo-yarn-workspaces');
 
 const config = createMetroConfiguration(__dirname);
 
+config.server = {
+  enhanceMiddleware: (middleware) => {
+    return (req, res, next) => {
+      const url = new URL(req.url, `http://${req.headers.host}`);
+      const platform = url.searchParams.get('platform');
+
+      // When an asset is imported outside the project root, it has wrong path on Android
+      // So we fix the path to correct one
+      if (platform === 'android' && /\/assets\/.+\.png\?.+$/.test(req.url)) {
```

Comment:
is it intentional that this code is only affecting png assets? also, should this be running only when `EXPO_BUNDLE_APP` is true?

Reply:
so far we've only used pngs - and it should be running during development as well!

Reply:
gotcha, makes sense. do you think it's worth doing this override for all asset types, so if others are added in the future it will work right away? (and also, i wonder if maybe we should add this file to .npmignore, since it seems to be only for internal use)

Reply:
yes - I think the regex would be simpler and would likely prevent foot guns 
also yes to nom ignore 👍 will update